src/test/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private final OperationHandler operationHandler = new PurchaseOperationHandler();
+
+    @Before
+    public void setUp() {
+        OperationHandler balanceHandler = new BalanceOperationHandler();
+        balanceHandler.calculate(new Transaction("banana", 30));
+        balanceHandler.calculate(new Transaction("apple", 30));
```

Comment:
better use put method to add data to your storage, in this test class you are testing PurchaseOperationHandler